QUARKUS-1936.md

```diff
@@ -0,0 +1,31 @@
+# QUARKUS-1936 Certify RHBQ on Eclipse Adoptium Builds
+
+This document is a high level test plan for certification of Red Hat build of Quarkus on Temurin JVM.
+
+## Scope of the testing
+
+### Impact on test suites and testing automation
+* Temurin JVM 11 and 17 need to be added as JDKs to Quarkus QE Jenkins
+  * Temurin JVM 11 and 17 seem to be updated timely on `/qa/tools` mount point, so that will be used for the Elektra
+    time frame.
+* Temurin JVM 11 and 17 have to be added as JVMs to Polarion
+* Bare metal and OpenShift jobs in extended platform testing need to be extended to be ran with both Temurin 11 and 17.
+* This affects JVM mode only.
+
+### Impact on resources
+* 1 medium executor for bare metal testing for 4 hours on Temurin 11
```

Comment:
we do not use "medium" flavor, I think we end up using large in the test-suite jobs